pepper-apis/src/main/resources/db-changes/schema/DDP-3709-file-upload.xml

```diff
@@ -23,6 +23,21 @@
             <column name="file_upload_guid" type="varchar(20)">
                 <constraints nullable="false" unique="true" uniqueConstraintName="file_upload_guid_uk"/>
             </column>
+            <column name="study_id" type="bigint">
```

Comment:
I think the study id will be an important piece of data down the line so I'm adding it in. It also helps narrow things down during data export.